tests/fr/homeassistant_HassTurnOff.yaml

```
@@ -9,27 +9,31 @@ tests:
       slots:
         name: lumière du plafond
         area: chambre
+    response: "lumière du plafond éteint"
```

To be compliant with the style guide, response should not contain the entity name. For now, I would correct the test, but the reply should be `lumière éteinte`

```suggestion
    response: "lumière du plafond éteinte"
```